.github/actions/amazon-linux-build-action/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Amazon Linux is used to build tegola_linux so the CGO requirements are linked correctly
+FROM amazonlinux:latest
```

Comment:
Should we be using the latest here? That is what got us in trouble with ubuntu?

Reply:
I debated this, precisely because of the `ubuntu-latest` issue. My thinking is that the point of this container is to build tegola_lambda using the most recent Amazon Linux to avoid any potential linking issues. I don't expect Amazon Linux to have breaking changes in the near future, but a version bump from say Amazon Linux 2 to Amazon Linux 3 *should* work automatically.